docs/expose.md

```diff
@@ -18,7 +18,7 @@ The following example deploys two pgBouncer nodes exposed through a LoadBalancer
 proxy:
   pgBouncer:
     replicas: 2
-    image: percona/percona-postgresql-operator:{{ release }}-ppg14-pgbouncer
+    image: percona/percona-postgresql-operator:{{ release }}-pgbouncer
```

Comment:
Hi. This will be official image from `percona/percona-pgbouncer` repository but not `percona/percona-postgresql-operator`. We will provide exact image name when we have versioning ready.